pkg/kubectl/cmd/get.go

```diff
@@ -69,6 +72,7 @@ func NewCmdGet(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 			err := RunGet(f, out, cmd, args)
 			cmdutil.CheckErr(err)
 		},
+		ValidArgs: validArgs,
```

Comment:
IIRC `pods/$NAME` format was added for xargs support, so it's mainly for power-users scripting anyway. Having bash completions suggest the `kubectl get pods $NAME` format sg. If/when cobra enforces validity, we can revisit this.